app/models/maths_and_physics/eligibility.rb

```diff
@@ -45,6 +45,7 @@ class Eligibility < ApplicationRecord
       on_or_after_cut_off_date: 1,
     }, _prefix: :awarded_qualified_status
 
+    has_one :claim, as: :eligibility, inverse_of: :eligibility
```

Comment:
It would be more consistent with your earlier commit "Switch AdminChecksPresenters to accept claim" if you changed the `…::EligibilityAdminAnswersPresenter` classes to accept a claim instead of an eligibility, instead of adding this new association so that you can get to a claim from an eligibility. What do you think?

Reply:
Consistent in one way (all the presenters would accept a claim), but at the expense of encapsulation. The non-"checks" presenters are all "eligibility" presenters. They are responsible for presenting an eligibility to a user. It would make for a strange API for them to accept a claim to do so. Besides which, in the final version of the code these presenters do not actually call any method on the claim itself, and instead only talk to the eligibility. This relationship primarily exists so that the eligibility can see which year academic claim was made and therefor can use that to determine the QTS cut-off that was used for the QTS question. More on this in the comment below...

Reply:
Makes sense - I hadn‘t noticed that in the end we‘re only calling methods on the eligibility anyway.